Sources/VitaminUIKit/Components/TextField/VitaminTextField.swift

```diff
@@ -648,6 +648,7 @@ public class DesignableUITextField: UITextField {
                 imageView.isUserInteractionEnabled = true
             }
             rightView = imageView
+            rightView?.frame = self.rightViewRect(forBounds: self.bounds)
```

Comment:
Could you just add a little comment above your line to explain why we do this?
Something like `Due to a keyboard animation issue, we need to fix the frame of the right icon`.

Reply:
Comment added 👍

Reply:
Thanks